src/verify-valid-url.js

```diff
@@ -48,3 +48,19 @@ exports.findUrlsToValidateInScopes = function (scopes) {
 
   return toValidateUrls;
 };
+
+exports.findUrlsToValidateInServices = function (services) {
+  const toValidateUrls = [];
+
+  for (let specifier in services) {
+    const address = services[specifier];
+
+    if (canVerify(specifier)) {
+      toValidateUrls.push(new URL(address, specifier).href);
```

Comment:
I am not 100% sure that this is correct behavior. It seems to be, based on section (1)(2)(5) in https://wicg.github.io/import-maps/#resolve-an-imports-match?

Reply:
Double checked, i think were ok.